src/etc/dhcp/dhclient-exit-hooks.d/static-dhclient-hook

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright (C) 2021 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+python3 - <<PYEND
+import os
+from time import sleep
+from vyos.util import call
+from vyos.util import get_all_vrfs
+
+# Wait for the system to be fully configured.
+while not os.path.isfile('/tmp/vyos-config-status'):
+    sleep(0.2)
```

Comment:
We cannot do this, because dhclient should work normally even before any of VyOS services, for example during early boot to provide access to remote configuration data sources.